CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Can you change the minimum version to 3.5? This is the newest version of cmake that is available through apt-get